AccessCheckoutDemo/AccessCheckoutDemo/RestrictedCardFlowViewController.swift

```diff
@@ -26,6 +26,7 @@ class RestrictedCardFlowViewController: UIViewController {
             .accessBaseUrl(accessBaseUrl)
             .validationDelegate(self)
             .acceptedCardBrands(["visa", "mastercard", "AMEX"])
+            .disabledPanFormatting()
```

Comment:
I think it's just a typo looking at your code and that it should be `disablePanFormatting` (without a 'd') ?

Reply:
good catch, updated